modules/roles_profiles/manifests/profiles/gecko_t_osx_1014_generic_worker_staging.pp

```diff
@@ -4,7 +4,9 @@
 
 class roles_profiles::profiles::gecko_t_osx_1014_generic_worker_staging {
 
-    require roles_profiles::profiles::cltbld_user
+    class { 'roles_profiles::profiles::cltbld_user':
+        autologin => false,
```

Comment:
If a worker is running mutliuser, I don't think we even need the `cltbld` user.  Generic worker will create its own users and set autologin accordingly.

Reply:
sounds fine to me. This was to ensure removing the cltbld user from the autologin, but is not necessary with generic worker setting up the autologin task user.